modules/jupyter/main.tf

```diff
@@ -113,24 +113,41 @@ resource "helm_release" "jupyterhub" {
   namespace        = var.namespace
   create_namespace = !contains(data.kubernetes_all_namespaces.allns.namespaces, var.namespace)
   cleanup_on_fail  = "true"
-  # timeout increased to support autopilot scaling resources, and give enough time to complete the deployment 
-  timeout          = 1200
+  # This timeout is sufficient and ensures terraform doesn't hang for 20 minutes on error.
```

Comment:
is the timeout sufficient for both AP and standard?

Reply:
@umeshkumhar @blackzlq wasnt the reason for this because cert provisioning takes 20 mins?

Reply:
This is where documentation is important. I was going off the original comment, which is incorrect, as Ray Autoscaler provisions on demand and not during deployment time.

Can we bump this later once IAP is fully implemented?